### ISSUE TITLE
Some MySQL integer column definitions result in TEXT fields in sqlite3

### ISSUE THREAD

**Issue by mark-nicholson** [techouse/mysql-to-sqlite3#14]
**Describe the bug**

I have a database in MySQL and in it I define a table this way:

```
+----------+--------------+------+-----+---------+----------------+
| Field    | Type         | Null | Key | Default | Extra          |
+----------+--------------+------+-----+---------+----------------+
| id       | int          | NO   | PRI | NULL    | auto_increment |
| number   | int unsigned | NO   |     | NULL    |                |
| job_name | varchar(64)  | NO   |     | NULL    |                |
| duration | int unsigned | NO   |     | NULL    |                |
| result   | varchar(16)  | NO   |     | NULL    |                |
| url      | varchar(200) | NO   |     | NULL    |                |
| when     | datetime(6)  | NO   |     | NULL    |                |
+----------+--------------+------+-----+---------+----------------+
```

After I run the conversion to sqlite3, when I look at the same table's schema I see this:

```
CREATE TABLE IF NOT EXISTS "JenkinsInfo_jenkinsjob" (
	"id" INTEGER NOT NULL,
	"number" TEXT NOT NULL,
	"job_name" VARCHAR(64) NOT NULL,
	"duration" TEXT NOT NULL,
	"result" VARCHAR(16) NOT NULL,
	"url" VARCHAR(200) NOT NULL,
	"when" DATETIME NOT NULL,
	PRIMARY KEY ("id")
);
```

**Expected behaviour**

Conversion of any "integer" types in MySQL should result in "integer" types within sqlite3.

After snooping some documentation, I see that sqlite3 does not really have an "unsigned int" whereas MySQL does.  Even so, signed vs unsigned is a much better situation than `int` vs `str`.  (... and in my case Python3's integers are all signed anyway)

**Actual result**

The `number` and `duration` columns in MySQL end up a "TEXT" columns in the sqlite3 -- which is not right.   

**System Information**

```bash
$ mysql2sqlite --version
```

```
| software               | version                                                              |
|------------------------|----------------------------------------------------------------------|
| mysql-to-sqlite3       | 1.3.7                                                                |
|                        |                                                                      |
| Operating System       | Linux 4.14.35-2025.403.3.el7uek.x86_64                               |
| Python                 | CPython 3.6.8                                                        |
| MySQL                  | mysql  Ver 8.0.22 for Linux on x86_64 (MySQL Community Server - GPL) |
| SQLite                 | 3.7.17                                                               |
|                        |                                                                      |
| click                  | 6.7                                                                  |
| mysql-connector-python | 8.0.23                                                               |
| python-slugify         | 4.0.1                                                                |
| pytimeparse            | 1.1.8                                                                |
| simplejson             | 3.17.2                                                               |
| six                    | 1.14.0                                                               |
| tabulate               | 0.8.9                                                                |
| tqdm                   | 4.58.0                                                               |
```

**Additional context**

I use MySQL to store the production database, then export a copy to play-around with in sqlite3 in a test environment.

In my code I get type errors (in Python) where the code expects the value to be an `int` but it is actually a `str`, and the issue is a result that the actual underlying database has defined the column as TEXT.

I run the tool this way (in a script):

```bash
mysql2sqlite \
	      --sqlite-file ${sqlite_db_name} \
	      --vacuum --quiet \
	      --mysql-database fajita_web \
	      --mysql-user "${sql_user}" --mysql-password "${sql_pwd}"
```

The example above is just one table of my larger database.  However, I do see this same problem in most of the tables.

**Comment by techouse** [comment]
Hey!

Thanks for the extensive bug report.

According to [this line here](https://github.com/techouse/mysql-to-sqlite3/blob/master/mysql_to_sqlite3/transporter.py#L201) it should handle `INT` as `INTEGER`. It could be that the addition of `UNSIGNED` is breaking it and defaulting to the [default which is text](https://github.com/techouse/mysql-to-sqlite3/blob/master/mysql_to_sqlite3/transporter.py#L205).

Let me investigate and see if this is the cause.

**Comment by techouse** [comment]
Fixed in [v1.3.8](https://github.com/techouse/mysql-to-sqlite3/releases/tag/v1.3.8)

**Comment by mark-nicholson** [comment]
Brilliant!  Thank you for the fix!  That does the job.
